### PR TITLE
Replace admin_static conditionaly

### PR DESCRIPTION
**Pull request by freezed**
With these settings : https://gist.github.com/freezed/d5b18e8218c23fc64e4e8dc467e4b2a4

See #213

### CODE REVIEW

**Comment by freezed** [comment]
This is working for you ?

**Comment by barseghyanartur** [comment]
@freezed:

I didn't test it yet. Just merged to test later.

The bad thing is that all pull requests have been made against master, which is development branch. Stable is v0.13.x, thus it will take much longer time for all the changes to become public.

**Comment by freezed** [comment]
Oups, sorry for using `master`… I'll switch if I had another changes.

**Comment by 0pt1c** [comment]
Is there any estimate when this code might be tested and released into a public release branch? We are trying to update to django 3.0 and we are getting errors from the templates with the deprecated import.

I also tried installing the `master` branch directly and still got the error when running django 3.0. It looks like the version check isn't working:

Screenshot from 2020-04-02 13-58-30

File: src/fobi/contrib/themes/simple/templates/simple/dashboard.html [hunk 1]
```diff
@@ -1,6 +1,14 @@
 {% extends fobi_theme.base_edit_template %}
 
-{% load i18n admin_static %}
+{% if VERSIONS.DJANGO_GTE_2_1 %}
+    {% load i18n static %}
+{% else %}
+    {% if VERSIONS.DJANGO_GTE_2_1 %}
+    {% load i18n static %}
```
Comment (0pt1c) [inline src/fobi/contrib/themes/simple/templates/simple/dashboard.html:7]: this should be removed

**Pull request merged**